ipykernel/inprocess/ipkernel.py

```diff
@@ -128,12 +129,10 @@ def _shell_class_default(self):
         return InProcessInteractiveShell
 
     def _stdout_default(self):
-        from ipykernel.iostream import OutStream
-        return OutStream(self.session, self.iopub_socket, u'stdout', pipe=False)
+        return OutStream(self.session, self.iopub_socket, u'stdout')
 
     def _stderr_default(self):
-        from ipykernel.iostream import OutStream
-        return OutStream(self.session, self.iopub_socket, u'stderr', pipe=False)
+        return OutStream(self.session, self.iopub_socket, u'stderr')
```

Comment:
These two are relying on the deprecated API of passing a socket rather than an IOPubThread instance, right?